lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -71,6 +71,23 @@ <h5 class="hd hd-5">${_("Rescore")}</h5>
 
     <br><br>
 
+    %if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS'):
+        <h5 class="hd hd-5">${_("Score Override")}</h5>
+        <label for="override-problem-score-single">${_("For the specified problem, override the learner's score.")}</label>
+        <br><br>
+        <label for="score-select-single">
+            ${_("New score for problem, out of the total points available for the problem *")}<br>
```

Comment:
I would LOVE for you to do that. Right now they don't really indicate required fields (it's not as if a user has to fill in all of them to accomplish a task on that page) and there's no explanation anywhere for what the asterisk means. I was going to go in and do this myself but if you want to, please go right ahead!